tokio/src/lib.rs

```diff
@@ -410,7 +410,9 @@ mod future;
 pub mod io;
 pub mod net;
 
+#[cfg(not(target_arch = "wasm32-wasi"))]
 mod loom;
+#[cfg(not(target_arch = "wasm32-wasi"))]
 mod park;
```

Comment:
Surely this does not compile. The `loom` module is required by lots of stuff in `tokio::sync`.

Reply:
It does compile and works, I've been using this project to test Tokio & Axum: https://github.com/rjzak/axum-wasm-test